src/session/session_dhandle.c

```diff
@@ -534,6 +552,7 @@ __wt_session_get_btree_ckpt(WT_SESSION_IMPL *session, const char *uri, const cha
 
         /* Look up the history store checkpoint. */
         if (hs_dhandlep != NULL) {
+
```

Comment:
Remove added newline.

Reply:
Removed in 2b6d19038ee81539fdce1f3c7a8e6470270fed54